sources/commands/option_env.c

```diff
@@ -0,0 +1,23 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   option_env.c                                       :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: inikulin <inikulin@student.42berlin.de>    +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2024/09/30 00:08:59 by inikulin          #+#    #+#             */
+/*   Updated: 2024/10/02 22:55:52 by inikulin         ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "commands.h"
+
+int	option_env(int choice, t_treenode *node, t_param *param, char **out)
+{
+	if (!choice)
```

Comment:
Instead of 
```
if (ft_strcmp(input, "echo") == 0) 
option_env(...)
```
I do
```
option_env (ft_strcmp(input, "echo") == 0, ...
...
int option_env(int choice...)
{
if (!choice)
return (0);
...
}
```
this way I save a line per each option in the caller function ([here](https://github.com/ioannnikulin/minishell/pull/53/files#diff-df9f1fc08f446de5e0e7f9df3d10d51e521923d282320e06bbb6ede2ef182aacR22))